src/main/java/mate/academy/bookstore/config/SecurityConfig.java

```diff
@@ -36,13 +36,13 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
             .authorizeHttpRequests(
                 auth -> auth
                     .requestMatchers(
```

Comment:
that are all what you need actually 
```suggestion
                    .requestMatchers(
                                        antMatcher("/auth/**"),
                                        antMatcher("/swagger-ui/**"),
                                        antMatcher("/v3/api-docs/**"))
                                .permitAll()
```